docs/docs/segment-os.md

```diff
@@ -25,6 +25,8 @@ Display OS specific info. Defaults to Icon.
 
 ## Properties
 
-- macos: `string` - the string to use for macOS - defaults to macOS icon
-- linux: `string` - the icon to use for Linux - defaults to Linux icon
-- windows: `string` - the icon to use for Windows - defaults to Windows icon
+- macos: `string` - the string/icon to use for macOS - defaults to macOS icon
+- linux: `string` - the string/icon to use for Linux - defaults to Linux icon
+- windows: `string` - the string/icon to use for Windows - defaults to Windows icon
+- wsl: `string` - the string/icon to use for WSL - defaults to WSL
```

Comment:
can we change this to ``defaults to `WSL` ``, same for adding the default below.